postgis/Makefile.in

```diff
@@ -91,6 +91,7 @@ PG_OBJS= \
 	lwgeom_geos_prepared.o \
 	lwgeom_geos_clean.o \
 	lwgeom_geos_relatematch.o \
+	lwgeom_hexgrid.o \
```

Comment:
As the file contains both squares and hexagons, maybe a different name would be better.

Reply:
done